.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+**Description**
+Include a summary of the change, and link the associated issue (if applicable)
+List any dependencies that are required for this change.
```

Comment:
Library dependencies and variables (with frequencies, units, etc). Also, separate for new and continuing PODs.